test/libsolidity/syntaxTests/constants/constant_transient_state_variable.sol

```diff
@@ -0,0 +1,5 @@
+contract C {
+    int constant public transient x = 0;
+}
```

Comment:
```suggestion
contract C {
    int constant public transient x = 0;
    int immutable public transient y;

    constructor() {
        y = 1;
    }
}
```

Add immutable coverage.

Reply:
There is already -> test/libsolidity/syntaxTests/immutable/state_var_transient_data_location.sol

Reply:
Ah neat, didn't notice we have those split in different dirs.